inc/massiveaction.class.php

```diff
@@ -387,8 +387,6 @@ function getCheckItem($POST) {
             if (isset($POST['check_items_id'])) {
                if (!$this->check_item->getFromDB($POST['check_items_id'])) {
                   exit();
-               } else {
```

Comment:
This part was very strange.
We have at choice, an exit on a bad id passed, or a forced getempty. So the getFromDB doesn't have any usage